tests/ImageSharp.Tests/IO/LocalFileSystemTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Six Labors.
+// Copyright (c) Six Labors.
```

Comment:
I don't know what this is about :/ maybe a different line ending or encoding?

Reply:
Don't worry about it.